support-frontend/assets/pages/supporter-plus-landing/twoStepPages/contributionsOnlyLanding.tsx

```diff
@@ -207,19 +208,22 @@ export function ContributionsOnlyLanding({
 					<PaymentFrequencyButtons
 						paymentFrequencies={paymentFrequencies.map(
```

Comment:
CARD to replace paymentFrequency with BillingPeriod from this component downwards

Reply:
Trello Card raised [here](https://trello.com/c/NmfEvi8t/1639-replace-payment-frequency-with-billingperiod)